test/ETO/Extras/ExtraTokenControllers.js

```diff
@@ -0,0 +1,352 @@
+            from: company,
+          });
+          await equityTokenController.executeForcedTransfer(investor1);
+          expect(await equityToken.balanceOf(investor1)).to.be.bignumber.eq(0);
+          expect(await equityToken.balanceOf(investor2)).to.be.bignumber.eq(inv1DistAmount);
+        });
+
+        it("should force transfer out of eto contract", async () => {
+          // company is allowed to force transfer tokens that were not claimed from commitment contract
+          // as mentioned: forced transfer make tokens non-trustless in the strict sense
+          // force transfer of remaining tokens
+          const amount = new web3.BigNumber(totalShares * (await equityToken.tokensPerShare())).sub(
+            inv1DistAmount,
+          );
+          await equityTokenController.enableForcedTransfer(
+            testCommitment.address,
+            investor2,
+            amount,
+            {
+              from: company,
+            },
+          );
+          await equityTokenController.executeForcedTransfer(testCommitment.address);
+          expect(await equityToken.balanceOf(testCommitment.address)).to.be.bignumber.eq(0);
+          expect(await equityToken.balanceOf(investor2)).to.be.bignumber.eq(amount);
+        });
+
+        it("should not unfreeze tokens", async () => {
+          let canTransfer = await equityTokenController.onTransfer(
+            investor1,
+            investor1,
+            investor2,
+            "1",
+          );
+          expect(canTransfer).to.be.false;
+          await equityTokenController.freezeHolder(investor1, { from: company });
+          await equityTokenController.unfreezeHolder(investor1, { from: company });
+          canTransfer = await equityTokenController.onTransfer(
+            investor1,
+            investor1,
+            investor2,
+            "1",
+          );
+          expect(canTransfer).to.be.false;
+        });
+      });
+    });
+
+    describe("RegDTransferController", async () => {
+      it("should deploy", async () => {
+        await deployController(RegDTransferController);
+        await prettyPrintGasCost("RegDTransferController deploy", equityTokenController);
+      });
+
+      it("should lock during lockin period", async () => {
+        // make investor 1 reg-d investor
+        const identityRegistry = await deployIdentityRegistry(universe, admin, admin);
+        // verified, requires reg d accreditiation, has accreditiation
+        await identityRegistry.setClaims(investor1, toBytes32("0x0"), toBytes32("0x31"), {
+          from: admin,
+        });
+        // simulate ETO
+        await deployController(RegDTransferController, { ENABLE_TRANSFERS_ON_SUCCESS: true });
+        await deployETO();
+        await makeInvestment(totalShares, inv1DistAmount);
+        // investor 2 is a regular investor with transfer rights
+        await testCommitment._distributeTokens(investor2, "1");
+        // should not be able to send
```

Comment:
maybe add ```// investor 1 should not be able to send```

this is really tiny detail, but like this the comment is more confusing than helping, because it reads as if the two comments are connected: 

"investor 2 is a regular investor AND should not be able to send

we can leave it though.... i just want to mention it